labelbox/client.py

```diff
@@ -1229,6 +1229,110 @@ def _format_failed_rows(rows: List[str],
                 )
             time.sleep(sleep_time)
 
+    def clear_global_keys(
+            self,
+            global_keys: List[str],
+            timeout_seconds=60) -> Dict[str, Union[str, List[Any]]]:
+        """
+        Gets data row ids for a list of global keys.
```

Comment:
Docstring should be updated

Reply:
yeah, I just copypasted for the format. We should work on an informative docstring.